recipes-bsp/bolt/bolt_%.bbappend

```diff
@@ -0,0 +1,23 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+DEPENDS += "update-rc.d-native"
+
+SRC_URI += "\
+   file://bolt \
+   file://boltctl \
+   file://0001-Update-udev-rule-for-hosts-that-don-t-have-systemd.patch \
+"
+
+FILES:${PN} += "${sysconfdir}/init.d/bolt"
+
+do_install:append() {
+   install -d ${D}${sysconfdir}/init.d
+   install -m 0755 ${WORKDIR}/bolt ${D}${sysconfdir}/init.d/bolt
+
+   # Move boltctl to a directory that's not in PATH and install a wrapper
+   # in its place
+   mv ${D}${bindir}/boltctl ${D}${libexecdir}/
+   install -m 0755 ${WORKDIR}/boltctl ${D}${bindir}/
```

Comment:
As long as we're messing with the binary paths: Is it sane for a non-privileged user to run this utility? If not, then we should consider putting it in `/sbin` instead.

Reply:
The utility can also be used for listing connected thunderbolt devices, their authorization status, etc. I think these operations should be allowed for non-`privileged` users also.

Access to system-configuration-changing-commands of the utility are already controlled by polkit (or dbus rules that I'm now testing).